lib/vpqr.js

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+const cborld = require('@digitalbazaar/cborld');
+const qrcode = require('@nuintun/qrcode');
```

Comment:
Use ESM style imports here. Also, move the destructuring from `qrcode` up into the import to enable tree-shaking.

```suggestion
const cborld = require('@digitalbazaar/cborld');
const qrcode = require('@nuintun/qrcode');
```

Reply:
Fixed.